lib/pascal_triangle.rb

```diff
@@ -0,0 +1,24 @@
+class PascalTriangle
+
+  def print_lines_to(line_number)
+    (0..line_number).map {|number| print_line(number)}
+  end
+
+  def print_line(line_number)
+    line = [1]
+    line_number.times {line = next_line(line)}
+    line
+  end
+
+  def next_line(previous_line)
+    new_line = []
+    new_line.push(1)
+    if previous_line.size >= 2
+      (0..previous_line.size-2).each {|index|
```

Comment:
I think it's uncommon to have this syntax using curly braces and then split it over two lines. If the line gets to long, then use 

[].each do |something|
    something bla bla
end